templates/kafka/0/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+zk:
```

Comment:
Nobody is immediately working on nesting but yes I expect we will add a way to address this fairly soon.

Is there some advantage to the static mode?  It doesn't sound like it from the way you describe...